tests/php/ORM/DataListTest.php

```diff
@@ -2518,6 +2498,25 @@ private function createEagerLoadData(): void
                     $manyManyThroughSubObj = new ManyManyThroughSubEagerLoadObject();
                     $manyManyThroughSubObj->Title = "manyManyThroughSubObj $i $j $k";
                     $manyManyThroughSubObj->write();
+
+                    // $results = DB::get_schema()->preparedQuery(
+                    //     "SELECT tablename FROM pg_catalog.pg_tables", []
+                    // );
+
+                    // foreach ($results as $result) {
+                    //     echo $result["tablename"] . "\n";
+                    // }
+
+                    // $results = DB::get_schema()->preparedQuery(
+                    //     "SELECT table_name, column_name, data_type FROM information_schema.columns WHERE data_type='tsvector' AND table_name like 'ManyManyThroughEagerLoadObjectManyManyThroughSubEagerLoadObject';", []
+                    // );
+
+                    // var_dump(DB::get_schema()->fieldList('ManyManyThroughEagerLoadObjectManyManyThroughSubEagerLoadObject'));
+
+                    // foreach ($results as $result) {
+                    //     var_dump($result);
+                    // }
+
```

Comment:
We can't just have a bunch of commented out cruft in here 😅

Reply:
Removed.